packages/shared/friends/sagas.ts

```diff
@@ -757,6 +770,9 @@ export async function getFriendRequestsProtocol(request: GetFriendRequestsPayloa
     // Get friends
     const friends: FriendsState = getPrivateMessaging(store.getState())
 
+    // Reject blockedUsers
+    await handleBlockedUsers(friends.fromFriendRequests)
```

Comment:
Log warning for users that could not be rejected in the background (below suggestion makes the return data available here).